Applications/DataHolderLib/Color.cpp

```diff
@@ -63,7 +73,7 @@ int readColorLookupTable(std::map<std::string, Color> &colors, const std::string
 
     return 1;
 }
-
+*/
```

Comment:
Is it possible to remove the commented code? If not, please add a comment why it is necessary to keep the code.